chore(connectors): update msteams documantation (#1616)
camunda/camunda-docs: docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md

| Expand response |                                                                 Get chat by id                                                                  |   Yes    |    dropdown     |                                                                                        Choose                                                                                        |
|     Filter      |                                                              List messages in chat                                                              |    No    |     string      | Sets the date range filter for the lastModifiedDateTime and createdDateTime properties. [Learn more about filtering](https://learn.microsoft.com/en-us/graph/filter-query-parameter) |
|   Message ID    |                                                               Get message in chat                                                               |   Yes    |     string      |                                                                           Microsoft teams chat message id                                                                            |

##### Expand response

> You can expand response for method **Get chat by ID**. For this in **Expand response** property you can :

Reviewer: I would suggest "For method  **Get chat by ID**, you can get more information in the response by using the dropdown property  **Expand response**. You can choose one of the following values: "

> You can expand response for method **Get chat by ID**. For this in **Expand response** property you can :

Reviewer 2: ```suggestion
You can expand response for method **Get chat by ID**. For this in the **Expand response** property you can:
```

> - choose **With chat members**, for getting information about members of chat;

Reviewer: - **With chat members**, to get information about chat members

> - choose **With chat members**, for getting information about members of chat;

Reviewer 2: ```suggestion
- select **With chat members**, for getting information about members of chat;
```

> - choose **With last message preview**, for getting last message in chat. This function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.

Reviewer: - **With last message preview**, to get last message in chat. This function doesn't work with [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type

> - choose **With last message preview**, for getting last message in chat. This function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.

Reviewer 2: ```suggestion
- select **With last message preview**, for getting last message in chat. **Note:** this function doesn't work for [client credentials type authentication](#client-credentials-type-authentication), make sure that you use another authentication type.
```

> - choose **Without expand**, for getting main information about chat.

Reviewer: - **Without expand**, to get main information about chat
